WiinUPro/Controls/DeviceStatus.xaml.cs

```diff
@@ -39,6 +39,8 @@ public UserControl Control
 
         public ImageSource Icon { get { return icon.Source; } }
 
+        private int extIndex = -1;

```

Comment:
In hindsight I'm not exactly sure why I decided to create the variable here and not inside UpdateType... I'll have to take a closer look at it later.